packages/deployments/contracts/contracts/core/connext/facets/BridgeFacet.sol

```diff
@@ -392,7 +389,9 @@ contract BridgeFacet is BaseConnextFacet {
   function bumpTransfer(bytes32 _transferId) external payable nonReentrant whenNotPaused {
     if (msg.value == 0) revert BridgeFacet__bumpTransfer_valueIsZero();
 
+    // TODO: should we store the fees or is there an easier way to reference whats been paid?
```

Comment:
only 5k for update here, at least. idk if there's anything else we can do here w/o adding sec assumptions

Reply:
maybe we should just track all the fees in the relayer fee vault if we're gonna transfer there anyway

Reply:
we can emit it in events and track it offchain, so ill remove